web/packages/teleport/src/components/Empty/Empty.tsx

```diff
@@ -105,23 +108,31 @@ export default function Empty(props: Props) {
           <Link
             to={{
               pathname: `${cfg.routes.root}/discover`,
-              state: { entity: resourceType },
+              state: {
+                entity:
+                  resourceType !== 'unified_resource' ? resourceType : null,
```

Comment:
Nit: wouldn't it be simpler just to allow `resourceType` to be `null` and get rid of this line? I think that "there's no particular resource type" conveys the idea better than "create an abstract type of resource that we only use when returning stuff to the UI".